src/main/java/ch/puzzle/eft/service/ExamFileService.java

```diff
@@ -80,4 +80,16 @@ protected File[] getSubjectDirectories() {
         return baseDir
                 .listFiles(File::isDirectory);
     }
+
+    public File getFileToDownload(String subjectName, String filename) {
+        File examToDownload = new File(getBasePath() + "/" + subjectName + "/" + filename);
+        if (!examToDownload
+                .exists()) {
+            logger
+                    .info("No file found for subject {} and filename {}", subjectName, filename);
+            throw new ResponseStatusException(HttpStatus.NOT_FOUND, String
+                    .format("Keine Unterordner im Pfad %s gefunden", subjectName));
```

Comment:
Are the logger message and the ResponseStatusException message different on purpose?